public/assets/js/allcount.js

```diff
@@ -54,6 +54,10 @@ allcountModule.config(["fieldRenderingServiceProvider", function (fieldRendering
             return s && moment(s, 'YYYY-MM-DD HH:mm:ss').toDate();
         }
 
+        function parseDateToMoment(s) {
```

Comment:
While testing I found that a Date wouldn't set the defaultDate correctly. However a moment would, I didn't investigate why.